analysis/make-epochs.py

```diff
@@ -0,0 +1,125 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: tutorialAlice
+#     language: python
+#     name: tutorialalice
+# ---
+
+# +
+from pathlib import Path
+import re
+
+import eelbrain
+import mne
+import trftools
+# -
+
+STIMULI = [str(i) for i in range(1, 13)]
+DATA_ROOT = Path("~").expanduser() / 'Data' / 'Alice'
+PREDICTOR_DIR = DATA_ROOT / 'predictors'
+EEG_DIR = DATA_ROOT / 'eeg'
+SUBJECTS = [path.name for path in EEG_DIR.iterdir() if re.match(r'S\d*', path.name)]
+# Define a target directory for epoched data and make sure the directory is created
+EPOCH_DIR = DATA_ROOT / 'Epochs'
+EPOCH_DIR.mkdir(exist_ok=True)
+
+tstart = -0.1
+tstop = 1
+
+# +
+# Load stimuli
+# ------------
+# Make sure to name the stimuli so that the TRFs can later be distinguished
```

Comment:
Did this code maybe survive from some copy-pasting? (comment about TRF names?) Why load all the stimuli here? For epoching all you need is the word onsets which should be in `AliceChapterOne-EEG.csv`, no?